app/models/manager_profile.rb

```diff
@@ -3,7 +3,8 @@ class ManagerProfile < ActiveRecord::Base
   has_many :events
 
   validates :company_name, presence: true
-  validates :subdomain, presence: true, uniqueness: true
+  validates :subdomain, presence: true, uniqueness: true,
+  :format => { with: /\A([a-zA-Z]+)/ }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.